vcpkg/concepts/glossary.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Glossary of vcpkg terms
+description: This article provides a glossary of common terms-of-art used in vcpkg's documentation.
+author: vicroms
+ms.author: viromer
+ms.date: 10/23/2023
+ms.prod: vcpkg
+ms.topic: glossary
+---
+
+# Glossary: vcpkg
+
+Some of the terms used in this documentation have similar meaning when discussed
+by the community. Because of that, and to avoid confusion and ambiguity. This
+article provides specific meaning to these terms. 
```

Comment:
```suggestion
Some of the terms used in this documentation have similar meanings when discussed
by the community. Because of that, and to avoid confusion and ambiguity. This
article provides specific meanings for these terms. 
```